src/main/scala/wdl4s/expression/WdlStandardLibraryFunctions.scala

```diff
@@ -114,6 +116,23 @@ trait WdlStandardLibraryFunctions extends WdlFunctions[WdlValue] {
     extractArguments flatMap arrayLength
   }
 
+  def prefix(params: Seq[Try[WdlValue]]): Try[WdlArray] = {
+    def extractTwoArguments: Try[(WdlValue, WdlValue)] = params.size match {
+      case 2 => (params.head |@| params.tail.head) map { (_, _) }
+      case n => Failure(new UnsupportedOperationException(s"prefix() expects two parameters but got $n"))
+    }
+
+    val makePrefixedString = (prefixString: WdlValue, strings: WdlValue) => (prefixString, strings) match {
+      case (WdlString(p), WdlArray(WdlArrayType(WdlStringType), ss)) =>
```

Comment:
Might be nice to support prefixing any primitive array type with:
```
case (WdlString(p), WdlArray(WdlArrayType(x), ss)) if x.isInstanceOf[WdlPrimitiveType] =>
```

NB: we'd still want Strings as output, obvs...